Modules/GLO/GLO_mnl_EPN.json

```diff
@@ -0,0 +1,118 @@
+{
+  "qc" : {
+    "config" : {
+      "database" : {
+        "implementation" : "CCDB",
+        "host" : "ali-qcdb.cern.ch:8083",
+        "username" : "not_applicable",
+        "password" : "not_applicable",
+        "name" : "not_applicable"
+      },
+      "Activity" : {
+        "number" : "42",
+        "type" : "2"
+      },
+      "monitoring" : {
+        "url" : "influxdb-unix:///tmp/telegraf.sock"
+      },
+      "consul" : {
+        "url" : "http://ali-consul.cern.ch:8500"
+      },
+      "conditionDB" : {
+        "url" : "http://localhost:8084"
+      },
+      "infologger" : { "" : "Configuration of the Infologger (optional).",
+                       "filterDiscardDebug" : "1",
+                       "" : "Set to true to discard debug and trace messages (default: false)",
+                       "filterDiscardLevel" : "21",
+                       "" : "Message at this level or above are discarded (default: 21 - Trace)" }
+    },
+    "tasks" : {
+      "Vertexing" : {
+        "active" : "true",
+        "className" : "o2::quality_control_modules::glo::VertexingQcTask",
+        "moduleName" : "QcGLO",
+        "detectorName" : "GLO",
+        "cycleDurationSeconds" : "600",
+        "maxNumberCycles" : "-1",
+        "dataSource" : {
+          "type" : "dataSamplingPolicy",
+          "name" : "VtxSampling"
+        },
+	"taskParameters" : {
+          "isMC": "false"
+        },
+        "location" : "local",
+	"localMachines": [
+            "epn",
+	    "localhost"
+        ],
+        "remoteMachine": "alio2-cr1-qc03.cern.ch",
```

Comment:
I will assign you `alio2-cr1-qc07.cern.ch`.

Reply:
Thanks.